docs/platform-packet.md

```diff
@@ -40,11 +40,17 @@ retry the boot typically fixes this.
 
 ## Boot
 
-LinuxKit on Packet boots the `kernel+initrd` output from moby
-via
-[iPXE](https://help.packet.net/technical/infrastructure/custom-ipxe). iPXE
-booting requires a HTTP server on which you can store your images. The
-`-base-url` option specifies the URL to the HTTP server.
+LinuxKit on Packet boots the `kernel+initrd` output from moby via
+[iPXE](https://help.packet.net/technical/infrastructure/custom-ipxe)
+which also requires a iPXE script. iPXE booting requires a HTTP server
+on which you can store your images. The `-base-url` option specifies
+the URL to the HTTP server. The boot process expects `<name>-kernel`,
+`<name>-initrd.img`, and `<name>-packet.ipxe` to be available at the
```

Comment:
I think it probably doesnt make sense to put "packet" in the name - there are other ipxe platforms and they generally work in exactly the same way, so a generic name is probably good and then we can reuse the code.

Reply:
I used to have `<name>.ipxe` but chose to include `packet` because some of the ipxe options may be specific. I can change it back...

Reply:
for example serial on `ttyS1` on x86